core/arch/arm32/sta/sub.mk

```diff
@@ -1,2 +1,7 @@
 srcs-y += sta_self_tests.c
 srcs-y += core_self_tests.c
+
+ifeq ($(WITH_SE_API),y)
```

Comment:
We might want to have se api for production, and don't want to have self test built-in. This is why this ifeq/endif here.